playwright-tests/tests/navigation.page.spec.ts

```diff
@@ -50,6 +50,13 @@ test('@NAV-005: Validate Find a mentor Button', async ({
   await basePage.clickElement(homePage.findMentorButton);
   await basePage.verifyURL('/mentorship/mentors');
 });
+
+  test('NAV-006 Validate logoclick navigation', async ({ basePage,homePage}) => {
+    await basePage.navigateToPath('/');
+    await expect(basePage.logo).toBeVisible();
```

Comment:
It is unnecessary to verify if the logo is visible, click action is enough. If logo is not present, it will throw an error.